JHUGenerator/main.F90

```diff
@@ -668,37 +668,41 @@ SUBROUTINE GetCommandlineArgs()
-    call ReadCommandLineArgument(arg, "ewp_L_E", success, ewp_L_E, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_R_E", success, ewp_R_E, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_L_M", success, ewp_L_M, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_R_M", success, ewp_R_M, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_L_T", success, ewp_L_T, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_R_T", success, ewp_R_T, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_L_U", success, ewp_L_U, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_R_U", success, ewp_R_U, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_L_C", success, ewp_L_C, success2=distinguish_HWWcouplings, success3=SetVprimeff)
-    call ReadCommandLineArgument(arg, "ewp_R_C", success, ewp_R_C, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_El_left", success, ewp_El_left, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_El_right", success, ewp_El_right, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Mu_left", success, ewp_Mu_left, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Mu_right", success, ewp_Mu_right, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Ta_left", success, ewp_Ta_left, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Ta_right", success, ewp_Ta_right, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Up_left", success, ewp_Up_left, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Up_right", success, ewp_Up_right, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Chm_left", success, ewp_Chm_left, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Chm_right", success, ewp_Chm_right, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Top_left", success, ewp_Top_left, success2=distinguish_HWWcouplings, success3=SetVprimeff)
+    call ReadCommandLineArgument(arg, "ewp_Top_right", success, ewp_Top_right, success2=distinguish_HWWcouplings, success3=SetVprimeff)
 
     call ReadCommandLineArgument(arg, "M_Vprime", success, M_Vprime, SetLastArgument, success2=UseVprime, success3=SetMVprime)
```

Comment:
This line also needs to be fixed. M_Vprime -> M_Zprime and M_Wprime separated. See https://github.com/JHUGen/JHUGen/blob/development/JHUGenMELAv2/MELA/test/testME_v2.c#L4972 for usage in WW decay, https://github.com/JHUGen/JHUGen/blob/development/JHUGenMELAv2/MELA/test/testME_v2.c#L1294 for VH and https://github.com/JHUGen/JHUGen/blob/development/JHUGenMELAv2/MELA/test/testME_v2.c#L1553 for VBF.